packages/cli/src/lib/webpack/render-html-plugin.js

```diff
@@ -45,13 +45,7 @@ module.exports = async function (config) {
 
 		// Unfortunately html-webpack-plugin expects a true file,
 		// so we'll create a temporary one.
-		const tmpDir = join(
-			os.tmpdir(),
-			`preact-cli-${Math.floor(Math.random() * 100000)}`
-		);
-		if (!existsSync(tmpDir)) {
-			mkdirSync(tmpDir);
-		}
+		const tmpDir = mkdtempSync(join(os.tmpdir(), 'preact-cli-'));
```

Comment:
Skipping on the collision handling as this generates (at least) 6 alphanumeric characters. It's pretty unlikely to ever collide